run_checks_and_format.py

```diff
@@ -40,13 +46,22 @@ def run_process(cmd_list):
 
 def repo_changes():
     """Check if repo files changed."""
-    out, err = run_process(['git', 'status', '--short'])
+    out, _err = run_process(['git', 'status', '--short'])
     out_lines = [l for l in out.split('\n') if l.strip()]
     return out_lines
 
 
 def run():
     """Run linters and formatters."""
+
+    # make sure scripts are available in environment paths
+    if sys.platform == 'win32':
```

Comment:
Why are we doing this here @ewerybody ?

Reply:
Well, I have multiple Python environments in parallel and the dev one with `pydocstyle`, `pycodestyle`, `yapf`, `isort` is not in the environment path. So these would not be available by just calling them right away.

Is there a better way to solve this? 🤔

Reply:
You will have to install either the linters on the same env as qtsass or viceversa. But this is a very specific hack for your setup that should not go in the package. Please remove it

Reply:
Hmm ... as you see in the code its just that the `Scripts` folder of the Python env I'm currently running is not in `PATH` env. So it does not matter where I have qtsass, does it?

... I would need a way to change the `Scripts` folder path in PATHs according to the Python env I wanna run... at best automatically. Maybe some scripts at first... 😔 well